docs/cli-reference/ks_registry_add.md

```diff
@@ -46,17 +42,15 @@ ks registry add <registry-name> <registry-uri> [flags]
 ks registry add databases github.com/example
 
 # Add a registry with the name 'databases' at the uri
-# 'github.com/example/tree/master/reg' and the version (branch name) 0.0.1
-# NOTE that "0.0.1" overrides the branch name in the URI ("master")
-ks registry add databases github.com/example/tree/master/reg --version=0.0.1
+# 'github.com/org/example/tree/0.0.1/registry' (0.0.1 is the branch name)
+ks registry add databases github.com/org/example/tree/0.0.1/registry --version=0.0.1`
```

Comment:
Missed removing `--version` from the example

Reply:
Good catch - updated.